core/senders/qq_batch_builder.py

```
@@ -0,0 +1,207 @@
+                if not should_exclude_text:
+                    if involved_channels and len(involved_channels) > 1:
+                        if not header_added and i == 0:
+                            add_header_this_time = True
+                            header_added = True
+                    else:
+                        if i == 0:
+                            add_header_this_time = True
+
+                if add_header_this_time:
+                    if text_parts:
+                        text_parts[0] = f"{header}\n​{text_parts[0]}"
```

Ruff (0.15.12)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)